app/src/main/java/com/kks/nimblesurveyjetpackcompose/util/PreferenceManager.kt

```diff
@@ -0,0 +1,61 @@
+package com.kks.nimblesurveyjetpackcompose.util
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import com.google.gson.Gson
+import com.kks.nimblesurveyjetpackcompose.R
+
+const val PREF_ACCESS_TOKEN = "access_token"
+const val PREF_REFRESH_TOKEN = "refresh_token"
+const val PREF_LOGGED_IN = "logged_in"
+
+class PreferenceManager(context: Context) {
+    private val preferences: SharedPreferences =
+        context.getSharedPreferences(
+            "pref_${context.getString(R.string.app_name)}",
+            Context.MODE_PRIVATE
+        )
+
+    fun setStringData(key: String, value: String) {
```

Comment:
We could make this nullable since `getStringData()` returns a nullable type 🤔 
```suggestion
    fun setStringData(key: String, value: String?) {
```